charts/coredns/Chart.yaml

```
@@ -19,5 +19,5 @@ maintainers:
 type: application
 annotations:
   artifacthub.io/changes: |
-    - kind: fix
-      description: Add default values to chart to fix cluster role name templating issue
+    - kind: feature
```

See https://artifacthub.io/docs/topics/annotations/helm/#supported-annotations for the allowed values

```suggestion
    - kind: changed
```

---

Ah thanks, updated it!